Header header

# Position on table, global, in meters
float64 pos_x
float64 pos_y

float64 yaw
